code/modules/jobs/job_types/roguetown/adventurer/types/combat/rare/swashbuckler.dm

```diff
@@ -6,49 +6,54 @@
 	allowed_sexes = list(MALE, FEMALE)
 	allowed_races = list(
 		"Humen",
-		"Elf",
 		"Half-Elf",
-		"Dwarf",
 		"Tiefling",
-		"Dark Elf"
+		"Aasimar",
+		"Half-Orc"
 	)
 	outfit = /datum/outfit/job/roguetown/adventurer/swashbuckler
 	maximum_possible_slots = 1
 	min_pq = 0
+	pickprob = 30
 	category_tags = list(CTAG_ADVENTURER)
 
 /datum/outfit/job/roguetown/adventurer/swashbuckler
+	head = /obj/item/clothing/head/roguetown/helmet/leather/headscarf
 	pants = /obj/item/clothing/under/roguetown/tights/sailor
 	belt = /obj/item/storage/belt/rogue/leather
 	armor = /obj/item/clothing/suit/roguetown/armor/leather/jacket/sea
 	backl = /obj/item/storage/backpack/rogue/satchel
-	backr = /obj/item/quiver/bolts
+	backpack_contents = list(/obj/item/natural/worms/leech = 2, /obj/item/storage/belt/rogue/pouch/coins/mid)
+	backr = /obj/item/fishingrod
```

Comment:
you'll wanna make this fishingrod/fisher or its gonna be missing line and bobber